sentry-apollo-3/src/main/java/io/sentry/apollo3/SentryApollo3Interceptor.kt

```diff
@@ -24,7 +23,6 @@ class SentryApollo3Interceptor : ApolloInterceptor {
     ): Flow<ApolloResponse<D>> {
         val builder = request.newBuilder()
             .addHttpHeader(SENTRY_APOLLO_3_OPERATION_TYPE, Base64.encodeToString(operationType(request).toByteArray(), Base64.NO_WRAP))
-            .addHttpHeader(SENTRY_APOLLO_3_OPERATION_NAME, Base64.encodeToString(request.operation.name().toByteArray(), Base64.NO_WRAP))
```

Comment:
@lbloder @adinauer do you recall why we have not used the Apollo one only directly?

Reply:
No real reason, other than that I seem to have overlooked them and didn't know apollo already provides these.
There was an issue around the headers though, with regards to special characters, which is why we now Base64 encode them (#2680). Hopefully this issue doesn't come back from using the Apollo Headers.

Reply:
The [issue](https://github.com/getsentry/sentry-java/issues/2680) is about the variables specifically which is the sentry custom header and not the one provided by Apollo, I will keep the base 64 for this.
I will use Apollo headers for operation name and id.